.circleci/config.yml

```diff
@@ -955,15 +955,18 @@ jobs:
           paths:
             - desktop\release
 
+  # Publish all the artifacts generated by the previous wp-desktop- jobs to a
+  # GitHub release and generate the release notes using the Git commits history
   wp-desktop-publish:
     docker:
       - image: circleci/golang:1.12
-    working_directory: /home/circleci/wp-calypso
```

Comment:
I'm surprised changing the path from `/home/circleci` to `~` is necessary to get this working.

For reference, [here](https://github.com/Automattic/wp-desktop/blob/2fbfbe63e834b6954a891a3bc5ab7686ba6171b5/.circleci/config.yml#L570-L613) is the identical job in the legacy wp-desktop repository.

Reply:
You are right. Turns out the only thing I needed was to add the `checkout` step _before_ the `attach_workspace: at:` one. See https://github.com/Automattic/wp-calypso/pull/48331.

It just took me a few experiments to figure it out and this branch still carried most of them.

I reverted to the minimum required set of changes now.

Reply:
Ah - yeah, adding the checkout step makes sense (so the git history will be available). Thank you for catching this, and for fixing! 🙇